hardhat.config.ts

```diff
@@ -10,7 +10,7 @@ import 'solidity-coverage'
 import 'tsconfig-paths/register'
 import { HardhatUserConfig } from 'hardhat/types'
 import { task } from 'hardhat/config'
-import { pathStore } from '@solomon/shared/util-path-store'
+import { pathStore } from './libs/shared/util-path-store/src/index'
```

Comment:
This makes sense, though it's unfortunate that TS config paths don't work properly in config files. We have had similar problems with some frontend config as well. I believe it's due to the environment the files are loaded in not being aware of tsconfig paths.